### PR TITLE
Added a README file

### PR DESCRIPTION
**Pull request by ritviksangwan13**
Added a README file for the project

### CODE REVIEW

**Comment by RitvikSangwan** [comment]
This README is too short. Can you elaborate more?

**Review by RitvikSangwan — CHANGES_REQUESTED** [review]
Appreciate your follow up here.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is the README guys.
+
+Added more description.
```
Comment (RitvikSangwan) [inline README.md]: Add more description here.

**Review by RitvikSangwan — CHANGES_REQUESTED** [review]
Make it more explanatory

File: README.md (hunk 1 above)
Comment (ritviksangwan13) [reply README.md]: I have added more description

**Review by RitvikSangwan — APPROVED** [review]

**Pull request merged**